.clang-format

```diff
@@ -83,5 +83,4 @@ SpacesInParentheses: false
 SpacesInSquareBrackets: false
 Standard:        Cpp11
 TabWidth:        8
-UseTab:          Never
-...
+UseTab:          Never
```

Comment:
Could we add a trailing linefeed at the end of this file (also .clang-tidy and third-party/.clang-tidy).

Reply:
Yes